src/App.css

```diff
@@ -1,39 +1,34 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.calc {
+  padding: 3px;
   display: flex;
-  flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  flex-direction: column;
 }
 
-.App-link {
-  color: #61dafb;
+.scrn {
+  height: 2.5rem;
+  border: none;
+  background-color: #b0b0b0;
+  width: 11.9rem;
+  text-align: right;
+  padding-top: 1.25rem;
+  padding-right: 0.5rem;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.btn {
+  width: 3rem;
```

Comment:
- [ ] [Optional] Nice job with styling it looks like the given design but it would be much better if you make the calculator buttons a little bit wider.

Reply:
Great Idea Thanks a lot @Omar-Muhamad  also  fixed the width of the '0'.